tests/Driver/MonolithUploadDriverTest.php

```diff
@@ -18,7 +18,7 @@ class MonolithUploadDriverTest extends TestCase
      */
     private $handler;
 
-    protected function setUp()
+    protected function setUp(): void
```

Comment:
Don't these void return type clash with your other PR (which has PHP 7.0 support)

Reply:
You are completely right. I have to use a specific version of orchestra/testbench as they added the void return type in their 3.6 version.